src/Ocelot/Cache/DefaultMemoryCache.cs

```diff
@@ -2,12 +2,12 @@
 
 namespace Ocelot.Cache
 {
-    public class AspMemoryCache<T> : IOcelotCache<T>
+    public class DefaultMemoryCache<T> : IOcelotCache<T>
     {
         private readonly IMemoryCache _memoryCache;
         private readonly Dictionary<string, List<string>> _regions;
```

Comment:
```suggestion
        private readonly IDictionary<string, List<string>> _regions;
```

Reply:
@RaynaldM Hey Ray!
The `_regions` is an internal private class member and is not visible to class clients. Additionally, this member is not initialized by the constructor's argument. I also don't see a benefit in changing to a more abstract type since the member isn't exposed for testing. It would be more logical to change the storage type for rapid concurrent operations (for thread safety), but as far as I know, `Dictionary<string, List<string>>` is [thread-safe](https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2?view=net-8.0#thread-safety), correct?